MyEnumerable.rb

```diff
@@ -0,0 +1,20 @@
+d# Create a module MyEnumerable that implements the following methods.
```

Comment:
- [ ] It would be better if we are able to run the code without any errors. Kindly remove the "d" in line 1
![Screenshot from 2022-07-21 17-46-10](https://user-images.githubusercontent.com/46388113/180279880-9f5b78a5-2e97-44da-9a08-f5751e402ed9.png)

Reply:
Fixed